docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
```

Comment:
@digitaldan,
I saw this at the other docker repo and I must honestly say I ve mixed feelings with that. Why do we want to force "docker" users to use Lets Encrypt? How about other ones already having certificates and dont need to generate new ones? Shouldnt this be the way people put their certs in a folder and they are used in container? As we dont have any lets encrypt in our guide..why do we should it go into the docker image. I am not against it and can add it..but iam not sure if it is the best fit for all..